src/pages/FavouriteMovie.js

```diff
@@ -55,10 +65,10 @@ class FavouriteMovie extends Component {
           </button>
         </div>
 
-        {this.state.movie_data.map((i) => (
-          <div className="div1">
-            <h4 className="par">{i.Title}</h4>
-            <img src={i.Poster} id="imageMovie"></img>
+        {this.state.movie_data.map((i, x1, x2) => (
+          <div key={x1} className="div1">
```

Comment:
Not sure what classNAme="div1" should mean.
Naming for style classes should be descriptive. And since this is a basic layout, you don't really need any custom classes, bootstrap classes should be sufficient for the styling needs here.